src/theory/arith/theory_arith_private.h

```diff
@@ -237,7 +237,7 @@ class TheoryArithPrivate {
   context::CDQueue<ArithVar> d_constantIntegerVariables;
 
   Node callDioSolver();
-  Node dioCutting();
+  TrustNode dioCutting();
```

Comment:
Can you give some quick documentation on the form of this lemma?

Reply:
Sure thing. Done in bf6184e25.